tests/beta/pydantic/schema/test_mutation.py

```diff
@@ -0,0 +1,83 @@
+import pydantic
+
+import strawberry
+
+
+def test_mutation():
+    class User(pydantic.BaseModel):
+        name: pydantic.constr(min_length=2)
+
+    @strawberry.beta.pydantic.input(User, fields=["name"])
+    class CreateUserInput:
+        pass
+
+    @strawberry.beta.pydantic.type(User, fields=["name"])
+    class UserType:
+        pass
+
+    @strawberry.type
+    class Query:
+        h: str
+
+    @strawberry.type
+    class Mutation:
+        @strawberry.mutation
+        def create_user(self, input: CreateUserInput) -> UserType:
+            return UserType(input.name)
```

Comment:
Don't you need to run `data = input.to_pydantic()` here?

Reply:
`to_pydantic` is needed if you also want to run validation

Reply:
👍